src/parsers/utils.hpp

```diff
@@ -52,6 +80,10 @@ namespace se3
     // else return an empty string that will provoke an error in loadPolyhedronFromResource()
     namespace bf = boost::filesystem;
 
+    // In case the mesh_path is such as "file://path/to/mesh", check for /path/to/mesh and return it if it exists
+    if ( bf::exists( bf::path(urdf_mesh_path.substr(6, urdf_mesh_path.size())) ))
+      return std::string(urdf_mesh_path.substr(6, urdf_mesh_path.size()));
```

Comment:
Then, looking for `://` in the file name and analysing what's before should be safe enough.

Boost Regex is not a pure template library. I don't know what it templated and what is not but I know the above code does not work without being linked to the boost regex library.